ironfish/src/account/account.ts

```diff
@@ -138,14 +138,13 @@ export class Account {
   }> {
     const unspentNotes = []
 
-    for (const [hash, { noteIndex, serializedNote, spent, transactionHash }] of this
-      .decryptedNotes) {
-      if (!spent) {
+    for (const [hash, decryptedNote] of this.decryptedNotes) {
+      if (!decryptedNote.spent) {
         unspentNotes.push({
           hash,
-          index: noteIndex,
-          note: new Note(serializedNote),
-          transactionHash,
+          index: decryptedNote.noteIndex,
+          note: new Note(decryptedNote.serializedNote),
+          transactionHash: decryptedNote.transactionHash,
```

Comment:
This occluded the `transactionHash` above. Did we mean to use this one, or did we mean to use the one passed in?

Reply:
I don't think there's anything passed in. It looks like that might be from the return type.

Reply:
oh true, opps.